Documentation.docc/V4_API_Migration_guide.md

```diff
@@ -32,18 +32,8 @@ You must manually add the autogenerated file we committed to the repository
 to your project, and `#import RevenueCat-Swift.h` in your bridging header. You can see how we do this in our
 [SPMIntegration project](https://github.com/RevenueCat/purchases-ios/tree/main/IntegrationTests/SPMIntegration).
 
-### StoreKit 2
-
-You can opt in to using `StoreKit 2` features when you configure `Purchases`:
-```swift
-Purchases.configure(
-	withAPIKey: "your_api_key",
-	appUserID: nil,
-	observerMode: false,
-	userDefaults: nil,
-	useStoreKit2IfAvailable: true
-)
-```
+### Automatic Migration
+When building your project using v4, Xcode should automatically provide one-click fixes methods and types that have been renamed. For the most part, the migration should be doable by just building and applying Xcode's automatic fix-its when they pop up.
```

Comment:
Maybe add a line like "if you encounter any problems, please file an issue"? Maybe we can improve things for RC2 that we haven't thought about.

Reply:
Good call. We have a line about it at the end of this doc, but I think it's probably a good idea to have more than one callout in the doc